index.html

```diff
@@ -108,59 +113,60 @@ <h2 class="p-title">
                 </li>
                 <li>2015</li>
             </ul>
-        </div>
 
-        <p class="about">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-        </p>
 
-        <ul class="languages">
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-        </ul>
-        <div><button class="clickme" type="button">See Projects</button></div>
-    </section>
+            <p class="about">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+            </p>
 
-    <section class="skills project-card">
+            <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <div><button class="clickme" type="button">See Projects</button></div>
+        </div>
+    </section>
+    <!--section4-->
+    <section class="project-card">
         <div class="image">
             <img src="./images/portfolio3.png" alt="Image3" />
-            <div class="stories">
-                <h2 class="p-title">Tonic</h2>
-                <ul class="subwork">
-                    <li class="work1">CANOPY</li>
-                    <li>
-                        <img src="./images/Counter.svg" alt="" />
-                    </li>
-                    <li>Back End Dev</li>
-                    <li>
-                        <img src="./images/Counter.svg" alt="" />
-                    </li>
-                    <li>2015</li>
-                </ul>
+        </div>
+        <div class="background">
+            <h2 class="p-title">Tonic</h2>
+            <ul class="subwork">
+                <li class="work1">CANOPY</li>
+                <li>
+                    <img src="./images/Counter.svg" alt="" />
+                </li>
+                <li>Back End Dev</li>
+                <li>
+                    <img src="./images/Counter.svg" alt="" />
+                </li>
+                <li>2015</li>
+            </ul>
 
-                <p class="about">
-                    A daily selection of privately personalized reads; no accounts or
-                    sign-ups required.
-                </p>
+            <p class="about">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+            </p>
```

Comment:
- Kindly change the fixed `width for the paragraph that takes a class `about` to `percent` instead of `px` because it causes general horizontal scrolling in the desktop version.
![image](https://user-images.githubusercontent.com/24703171/155774506-31a7273a-daa4-4ca4-b279-c09d531b29e1.png)